### PR TITLE
All the docs

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -16,24 +16,68 @@
...
+    subprocess.call('cd ../ && doxygen doc/doxygen-java.conf', shell=True)
 
 # -- General configuration ------------------------------------------------
 
+import semver
+
+# -- Project information -----------------------------------------------------
+
+project = 'deepspeech-api'
+copyright = '2019, DeepSpeech Team'
```
Comment (reuben) [inline doc/conf.py]: nit: Mozilla Corporation?

File: doc/conf.py [hunk 2]
```diff
@@ -52,15 +96,6 @@
 copyright = u'2017, Mozilla Research'
 author = u'Mozilla Research'
```
Comment (reuben) [inline doc/conf.py]: Should probably update copyright and author here too.

File: doc/index.rst [hunk 3]
```diff
@@ -8,42 +8,36 @@ Welcome to DeepSpeech's documentation!
 
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
+   :caption: Inroduction
```
Comment (reuben) [inline doc/index.rst]: The long form documentation (.rst files) are very out-of-date. For example, they still talk about a bidirectional LSTM. We should either update them or remove them entirely.

Comment (lissyx) [reply doc/index.rst]: Your call. I was thinking we could update them after, for example.

File: doc/conf.py (hunk 2 above)
Comment (lissyx) [reply doc/conf.py]: Sure, what do you want here?

File: doc/conf.py (hunk 1 above)
Comment (lissyx) [reply doc/conf.py]: copyright = MoCo ?

Comment (reuben) [reply doc/conf.py]: That's my understanding of our work contract, yes :P

Comment (reuben) [reply doc/conf.py]: Dunno if France has different rules.

File: doc/conf.py (hunk 2 above)
Comment (reuben) [reply doc/conf.py]: Matching above, so 2019 Mozilla Corporation

File: doc/index.rst (hunk 3 above)
Comment (reuben) [reply doc/index.rst]: OK. I'll file an issue

File: doc/conf.py (hunk 2 above)
Comment (lissyx) [reply doc/conf.py]: Indeed that part is just useless, it's overwriting previous values

**Pull request merged**